dts/bindings/display/led-strip-matrix.yaml

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2024 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Generic LED strip matrix (LED strip arranged in a grid pattern)
+
+compatible: "led-strip-matrix"
+
+include: display-controller.yaml
+
+properties:
+  circulative:
+    type: boolean
+    description: |
+      Use a circulative layout that returns to the left edge of the next row
+      after reaching the right edge.
+      If not set, turn around and go left in a serpentine layout when it reaches
+      the right edge.
+
+      * circulative layout
+      [ 0][ 1][ 2][ 3]
+      [ 4][ 5][ 6][ 7]
+      [ 8][ 9][10][11]
+      [12][13][14][15]
+
+      * serpentine layout
+      [ 0][ 1][ 2][ 3]
+      [ 7][ 6][ 5][ 4]
+      [ 8][ 9][10][11]
+      [15][14][13][12]
+
+  start-from-right:
+    type: boolean
+    description: |
+      Specify if the first LED is at the top right.
+
+      * Start from the right with a serpentine layout
+        [ 3][ 2][ 1][ 0]
+        [ 4][ 5][ 6][ 7]
+        [11][10][ 9][ 8]
+        [12][13][14][15]
+
+      * Start from the right with a circulative layout
+        [ 3][ 2][ 1][ 0]
+        [ 7][ 6][ 5][ 4]
+        [11][10][ 9][ 8]
+        [15][14][13][12]
+
+  width:
+    description: |
+      Specifies the overall `width` of the matrix.
+      If the matrix consists of multiple modules, it is the sum of their widths.
+
+  height:
+    description: |
+      Specifies the overall height of the matrix.
+      If the matrix consists of multiple modules, it is the sum of their heights.
+
+  horizontal-modules:
+    type: int
+    default: 1
+    description: |
+      If the display forms with multiple modules,
+      specify the horizontal number of modules.
+      The number must be able to divide the width value.
+      If not set, it controls a single matrix.
+
+      * Modules are arranged circulative from top left to bottom right.
+        [M0][M1]
+        [M2][M3]
```

Comment:
Since I was expecting a module like [this](https://www.amazon.com/dp/B0CG64NQ75) that the user could connect with the connector, I didn't give much thought to the arrangement of the modules. However, the serpentine arrangement can have shorter wiring, so I will consider it.